pychromecast/controllers/spotify.py

```diff
@@ -11,10 +11,9 @@
 APP_NAMESPACE = "urn:x-cast:com.spotify.chromecast.secure.v1"
 TYPE_GET_INFO = "getInfo"
 TYPE_GET_INFO_RESPONSE = "getInfoResponse"
-TYPE_SET_CREDENTIALS = "setCredentials"
-TYPE_SET_CREDENTIALS_ERROR = "setCredentialsError"
-TYPE_SET_CREDENTIALS_RESPONSE = "setCredentialsResponse"
-
+TYPE_ADD_USER = "addUser"
```

Comment:
Can you describe what this new api call does?

Reply:
As far as we can tell it does the same thing that `setCredentials` used to do, with the payload in a slightly different format.